apis/v1alpha3/backendtlspolicy_types.go

```diff
@@ -144,10 +168,11 @@ type BackendTLSPolicyValidation struct {
 	//
 	// If WellKnownCACertificates is unspecified or empty (""), then CACertificateRefs
 	// must be specified with at least one entry for a valid configuration. Only one of
-	// CACertificateRefs or WellKnownCACertificates may be specified, not both. If an
-	// implementation does not support the WellKnownCACertificates field or the value
-	// supplied is not supported, the Status Conditions on the Policy MUST be
-	// updated to include an Accepted: False Condition with Reason: Invalid.
+	// CACertificateRefs or WellKnownCACertificates may be specified, not both.
+	// If an implementation does not support the WellKnownCACertificates field, or
+	// the supplied value is not recognized, the implementation MUST ensure the
+	// `Accepted` Condition on the BackendTLSPolicy is set to `status: False`, with
+	// a Reason `Invalid`.
```

Comment:
Any reason these don't appear in the consts?  Also, a better reason might be "InvalidWellKnownCACertificates".

Reply:
I’m trying to minimize changes at this late stage, this line was only moved.

The consts are defined in https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha2/policy_types.go.